packages/text/src/Text.ts

```diff
@@ -165,6 +165,11 @@ export class Text extends Sprite
     {
         const style = this._style;
 
+        if (!style)
```

Comment:
I get the `updateText` in the `set text` method, but why the check for no style here?

Reply:
![image](https://user-images.githubusercontent.com/6738986/86855426-e11fe480-c0ec-11ea-9203-f2ceb85b5d12.png)
`this.text` triggers the `set` before `this.style`

Reply:
Maybe I can switch their places